### PR TITLE
Remove Promise global augmentation

### PR DESCRIPTION
**Pull request by IlyaKhD**

### CODE REVIEW

File: ts/dx.all.d.ts [hunk 1]
```diff
@@ -1852,6 +1850,10 @@ declare module DevExpress.data {
          * [descr:ODataStore.createQuery(loadOptions)]
          */
         createQuery(loadOptions: any): any;
+        /**
+         * [descr:ODataStore.insert(values)]
+         */
+        insert(values: any): Promise<any> & JQueryPromise<any>;
```
Comment (IlyaKhD) [inline ts/dx.all.d.ts:1856]: Known issue:
```suggestion
        insert(values: any): PromiseExt<any> & JQueryPromise<any>;
```
This will be fixed with another PR

**Review by churkin — APPROVED** [review]
